bundles/CoreBundle/src/Migrations/Version20230111074323.php

```diff
@@ -37,6 +37,9 @@ public function up(Schema $schema): void
             SettingsStore::set('BUNDLE_INSTALLED__Pimcore\\Bundle\\WordExportBundle\\PimcoreWordExportBundle', true, SettingsStore::TYPE_BOOLEAN, 'pimcore');
         }
 
+        // Delete this definition if it already exists, because upgrades from pimcore 10 sometimes get stuck in this state. See #14995
+        $this->addSql("DELETE FROM `users_permission_definitions` WHERE `key` = 'word_export'");
+
         // Append to the comma separated list whenever the permissions text field has 'translation' but not already word_export
         $this->addSql("INSERT INTO `users_permission_definitions` (`key`, `category`) VALUES ('word_export', 'Pimcore Word Export Bundle')");
```

Comment:
```suggestion
        $this->addSql("INSERT IGNORE INTO `users_permission_definitions` (`key`, `category`) VALUES ('word_export', 'Pimcore Word Export Bundle')");
```

Hey @IronSean thanks for the PR. We could simply go for a insert ignore here instead of deleting and readding the the same permission. 

This would create the problem though that the category is not updated anymore.
Maybe we could split it into two statements, a insert ignore to add the permission and an update to add the category?

Reply:
Thanks @mattamon, those are both cleaner. I think the latter better represents the intended end state (guaranteeing the column contains that value after the migration has run). I've updated it.